src/ontology/cl-edit.owl

```diff
@@ -30663,6 +30664,17 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:29904965"^^xsd:string) o
 AnnotationAssertion(rdfs:label obo:CL_4032001 "reelin GABAergic cortical interneuron")
 EquivalentClasses(obo:CL_4032001 ObjectIntersectionOf(obo:CL_0000099 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0000956) ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0061534) ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_000013879)))
 
+# Class: obo:CL_4033014 (peg cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "http://purl.obolibrary.org/obo/BTO_0005588"^^xsd:string) obo:IAO_0000115 obo:CL_4033014 "A non-ciliated secretory epithelial cell that is part of some oviduct.")
+AnnotationAssertion(<http://purl.org/dc/terms/date> obo:CL_4033014 "2023-02-02T15:07:12Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:22911892"^^xsd:string) oboInOwl:hasExactSynonym obo:CL_4033014 "intercalary cell")
+AnnotationAssertion(rdfs:label obo:CL_4033014 "peg cell")
+SubClassOf(obo:CL_4033014 obo:CL_0000151)
+SubClassOf(obo:CL_4033014 obo:CL_1001591)
```

Comment:
Is there evidence to support this assertion?

Reply:
Isn't this definitional?  'fallopian tube epithelium' = 'oviduct epithelium' in Uberon.  This seems reasonable given: 

<img width="656" alt="image" src="https://user-images.githubusercontent.com/112839/216591164-c64f6cf4-edcf-4bc1-be6d-9cbdc1f084b5.png"> https://en.wikipedia.org/wiki/Fallopian_tube

However, Uberon structure looks quite broken here as the term appears to cover many different epithelia of the female reproductive system.  Needs to be an Uberon ticket on this. Might be complicated due to cross-species nature of ontology.

Reply:
As noted above, PMID:22911892 seems to only investigate the fallopian tube epithelium as opposed to whole oviduct. Is there a reference (other than BTO) that reports this cell in other locations of the oviduct?